docs/web/contribute.md

```diff
@@ -43,88 +86,163 @@ assign the issue to yourself. This way, nobody else will work on it at the same
+follows [that of the Angular
+community](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-format),
+for maximum automation and consistency no matter who's contributing.
 
-## Commit philosophy
+**NOTE:** Ensure that you have run `pre-commit run` **before** running `cz`, otherwise
+you may fail the pre-commit checks and `cz` will not remember what you've typed.
 
-We aim to make our individual commits small and tightly focused on the feature
-they are implementing. If you find yourself making functional changes to
-different areas of the codebase, we prefer you break up your changes into
-separate Pull Requests. In general, a philosophy of one Github Issue per
-Pull Request is a good rule of thumb, though that isn't always possible.
+You'll want to use `cz` to make it easy to adhere to this structure without much pain.
+
+`cz` is already configured and ready to go if you're in a `nix-shell`, so just stage your changes and run `cz`:
+
+        :::sh
+        $ git add .
+        $ cz
 
-We avoid merge commits (and in fact they are disabled in the Github repository)
-so you may be asked to rebase your changes on top of the latest commits to
-master if there have been changes since you last updated a Pull Request.
-Rebasing your changes is usually as simple as running
-``git pull upstream master --rebase`` and then force-pushing to your branch:
-``git push origin <branch-name> -f``.
+You should see a series of (hopefully informative) prompts about what action to take next:
 
+1. Select the type of change you're committing. In this case, we're committing a bug fix, so we'll select fix:
 
-## Commit/PR messages
+        :::sh
+        ? Select the type of change that you're committing:
+          feat:     A new feature
+        ❯ fix:      A bug fix
+          docs:     Documentation only changes
+          style:    Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
+          refactor: A code change that neither fixes a bug nor adds a feature
+          perf:     A code change that improves performance
+          test:     Adding missing tests or correcting existing tests
+        (Move up and down to reveal more choices) 
 
-Well-structed commit messages allow us to generate comprehensive release notes
-and make it very easy to understand what a commit/PR contributes to our
-codebase. Commit messages and PR titles should be prefixed with a standard
-code the states what kind of change it is. They fall broadly into 3 categories:
-``FEAT (feature)``, ``BUG (bug)``, and ``SUPP (support)``. The ``SUPP``
-category has some more fine-grained aliases that you can use, such as ``BLD``
-(build), ``CI`` (continuous integration), ``DOC`` (documentation), ``TST``
-(testing), and ``RLS`` (releases).
+   Generally you don't need to think too hard about what category to select, but note that:
 
+   * `feat` will cause a minor version bump
```

Comment:
umm what?

Reply:
Please read this https://github.com/semantic-release/semantic-release.

It would also be helpful to articulate a question, since just "what?" does give me a whole lot to go on 🤣